tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -646,6 +687,217 @@ HRESULT STDMETHODCALLTYPE CorProfiler::ModuleLoadFinished(ModuleID module_id, HR
     {
         module_ids_.push_back(module_id);
 
+        // Scan module for [Trace] methods
+        mdTypeDef typeDef = mdTypeDefNil;
+        mdTypeRef typeRef = mdTypeRefNil;
+        static shared::WSTRING traceAttribute = WStr("Datadog.Trace.TraceAnnotations.TraceAttribute");
```

Comment:
typo: we changed the name 🙂 
```suggestion
        static shared::WSTRING traceAttribute = WStr("Datadog.Trace.Annotations.TraceAttribute");
```

Reply:
This is also defined down in `TypeNameMatchesTraceAttribute` - we should probably only define it once if possible, right?

Reply:
Yeah you're absolutely right, I'll unify those. Thanks for spotting that, I thought I'd already taken care of it 😅

Reply:
Done in 46ae5f8dd067fd13db2b590f9a283f7ed6c1edea

Reply:
Done in 46ae5f8dd067fd13db2b590f9a283f7ed6c1edea